src/main/kotlin/pet/project/app/dto/user/UserMapper.kt

```diff
@@ -0,0 +1,23 @@
+package pet.project.app.dto.user
+
+import pet.project.app.model.User
+
+object UserMapper {
+
+    fun RequestUserDto.toModel(): User {
+        return User(
+            this.login,
+            this.contact,
+            this.bookWishList
```

Comment:
[nit] this can be omitted in Kotlin
```suggestion
           login,
           contact,
           bookWishList
```

Reply:
Fixed: updated as suggested